### ISSUE TITLE
Update to support latest version of WordPress

### ISSUE THREAD

**Issue by karmatosed** [bigcommerce/bigcommerce-for-wordpress#458]
### Expected behavior
The plugin should not give a warning or have indication it doesn't support the latest version of WordPress. It should as of yesterday be tested and updated to work with version 6.6.

### Actual behavior
Currently both on the WordPress.org repo and also in the plugin directory you get the following in various formats. It is only saying 6.3.45.

<img width="838" alt="image" src="https://github.com/user-attachments/assets/d0f540a5-24a5-469c-b215-e4d771abda1a">

### Steps to reproduce behavior
You can see this either through installing the plugin or searching for it. Which means it looks like it potentially might not work. There is on the plugin directory a banner calling out this issue on the plugin.

### Workaround or possible solution
The recommendation is to follow a few steps:

1. Update the readme to support the latest version.
2. Test on the latest version.
3. Release a version of the plugin with the new information included.

This does mean an opportunity to do a new release but a priority at this point should be to get the removal of the message and the 'latest supported version' being several versions behind.

**Comment by karmatosed** [comment]
Simple step testing of plugin on WordPress 6.6

I took the plugin from download at Wordpress.org plugin directory through the admin dashboard to see in WordPress.org what might be an issue in preparation for this. If there are automated tests even better but I wanted to do a usability check just to see if simply changing the supported version was all we needed to do.

- [ ] Able to activate plugin
- [ ] Able to see welcome page to create account
- [ ] Able to sign up a new account
- [ ] Authenticate and make sure setup.

I got to point where there seemed to be lack of authentication. This might be as I don't have a full account or plan. It also showed me a curious error message or what I presume is one with a blank box. See this screenshot:

<img width="1610" alt="SCR-20240717-mhxs" src="https://github.com/user-attachments/assets/91ccbc5e-3e5f-4fa3-a309-644e5ffcc448">

For context, I have gone through and signed up, authenticated but I might be out of trial plan time.
